scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/goals/Goal.java

```diff
@@ -283,7 +284,8 @@ public String getName() {
    * @param simulationResults
    * @return a list of issues in the plan that diminish goal satisfaction
    */
-  public java.util.Collection<Conflict> getConflicts(Plan plan, final SimulationResults simulationResults) {
+  public java.util.Collection<Conflict> getConflicts(Plan plan, final SimulationResults simulationResults, final
+                                                     EvaluationEnvironment evaluationEnvironment) {
```

Comment:
Minor: The way this line got broken up is really weird.
```suggestion
  public java.util.Collection<Conflict> getConflicts(
    Plan plan, 
    final SimulationResults simulationResults, 
    final EvaluationEnvironment evaluationEnvironment
  ) {
```